### PR TITLE
Update playpanel

### PR DESCRIPTION
**Pull request by AnandHemachandran**
Updated the playpanel to show decibel value and reworked on the UI.

### CODE REVIEW

**Comment by anatoly-os** [comment]
Hi! Thank you for the contribution. Did you sign the CLA btw?

**Comment by AnandHemachandran** [comment]
Yes

**Comment by anatoly-os** [comment]
Could you please squash the commits and force push changes after that?
https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/git-workflow#squash

**Comment by AnandHemachandran** [comment]
Is this still valid?

**Comment by anatoly-os** [comment]
I should have tested it earlier. Will take a look. Thanks for the ping.

**Comment by anatoly-os** [comment]
Just built the PR. The volume slider shows crazy numbers:
screencast 2019-02-21 10-48-51

**Comment by AnandHemachandran** [comment]
ezgif com-crop
Updated the function volLabel() to stop the sudden shifting of decibel values.
Negative values are due to the different calibration or intial reference. 
Link to reference: https://stackoverflow.com/questions/2445756/how-can-i-calculate-audio-db-level

**Comment by anatoly-os** [comment]
@AnandHemachandran could you squash the commits please?

File: mscore/playpanel.cpp [hunk 1]
```diff
@@ -239,13 +242,16 @@ void PlayPanel::setGain(float val)
...
 
+
 //---------------------------------------------------------
 //   volumeChanged
 //---------------------------------------------------------
 
 void PlayPanel::volumeChanged(double val, int)
       {
       emit gainChange(val);
+      vol=val;
```
Comment (anatoly-os) [inline mscore/playpanel.cpp]: Spaces around the `=` are needed.

File: mscore/playpanel.cpp [hunk 2]
```diff
@@ -333,6 +339,19 @@ void PlayPanel::tempoSliderPressed(int)
...
       tempoSliderIsPressed = true;
       }
+//---------------------------------------------------------
+//   setVolume
+//---------------------------------------------------------
+      
+void PlayPanel::volLabel()
+      {
+      qDebug()<<"Volume value ="<<vol;
+      if (vol < 0.001)
```
Comment (anatoly-os) [inline mscore/playpanel.cpp]: Magic number? Put it in separate variable.

File: mscore/playpanel.cpp [hunk 3]
```diff
@@ -333,6 +339,19 @@ void PlayPanel::tempoSliderPressed(int)
       {
       tempoSliderIsPressed = true;
       }
+//---------------------------------------------------------
+//   setVolume
+//---------------------------------------------------------
+      
+void PlayPanel::volLabel()
+      {
+      qDebug()<<"Volume value ="<<vol;
```
Comment (anatoly-os) [inline mscore/playpanel.cpp]: Spaces around `<<` are needed.

File: mscore/playpanel.cpp [hunk 4]
```diff
@@ -333,6 +339,19 @@ void PlayPanel::tempoSliderPressed(int)
...
+      if (vol < 0.001)
+            vol=qreal(-200);
```
Comment (anatoly-os) [inline mscore/playpanel.cpp]: Another magic number? This one leads to a bug when you open play panel first time:
Untitled 2019-03-18 15-09-46

File: mscore/playpanel.cpp [hunk 5]
```diff
@@ -333,6 +339,19 @@ void PlayPanel::tempoSliderPressed(int)
...
+      else
+            vol=qreal(20.0) * std::log10(vol);
```
Comment (anatoly-os) [inline mscore/playpanel.cpp]: 20? Please use constexpr variables instead.

**Comment by AnandHemachandran** [comment]
I have fixed the issues that were mentioned above, Can you please review it.

**Comment by anatoly-os** [comment]
@AnandHemachandran great! Thank you for the fixes.

**Pull request merged**